micro_sam/training/training.py

```diff
@@ -93,11 +92,13 @@ def train_sam(
     freeze: Optional[List[str]] = None,
     device: Optional[Union[str, torch.device]] = None,
     lr: float = 1e-5,
-    scheduler: Optional[_LRScheduler] = None,
     n_sub_iteration: int = 8,
     save_root: Optional[Union[str, os.PathLike]] = None,
     mask_prob: float = 0.5,
     n_iterations: Optional[int] = None,
+    scheduler_kwargs: Optional[Dict[str, Any]] = None,
+    export_path: Optional[Union[str, os.PathLike]] = None,
```

Comment:
I don't want to integrate export here, because we will have more export options once the modelzoo export is integrated.